models/intel/vehicle-detection-0203/accuracy-check.yml

```diff
@@ -0,0 +1,21 @@
+models:
+  - name: vehicle-detection-0203
+
+    launchers:
+      - framework: dlsdk
+        adapter:
+           type: class_agnostic_detection
+
+    datasets:
+      - name: crossroad_extra_untagged_vehicle
```

Comment:
Labels are numbered from zero in this dataset, but in the prediction, labels are equal to one. Please add a new dataset(suggestion below) and use it in these cases.

` 
 - name: crossroad_extra_untagged_vehicle_labels_from_1
    data_source: crossroad_extra_untagged/val_data
    annotation_conversion:
      converter: mscoco_detection
      annotation_file: crossroad_extra_untagged/val_coco_no_bg_vehicle.json
      has_background: True
    annotation: crossroad_extra_untagged_vehicle_labels_from_1.pickle
    dataset_meta: crossroad_extra_untagged_vehicle_labels_from_1.json
`